app/helpers/application_helper.rb

```diff
@@ -193,6 +193,13 @@ def copyright_notice
     "#{start_year} - #{current_year}"
   end
 
+  def collection_link(collection, options = {})
```

Comment:
you should be able to fix the Codeclimate violation by using `**kwargs` here and at the last line:

`link_to(body, collection.path, **kwags)`

Reply:
Ok, changing this! :slightly_smiling_face: